src/main/java/uk/gov/justice/laa/crime/cfecrime/utils/FullMeansTestOutcomeCalculator.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.justice.laa.crime.cfecrime.utils;
+
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.FullAssessmentResult;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.MagCourtOutcome;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.CaseType;
+import uk.gov.justice.laa.crime.cfecrime.cma.enums.MeansTestOutcome;
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+public class FullMeansTestOutcomeCalculator {
+
+    public static MeansTestOutcome getFullMeansTestOutcome(FullAssessmentResult result, CaseType caseType, MagCourtOutcome magCourtOutcome){
+        log.debug("Get the outcome of the Full Means Test result = {} caseType = {}", result, caseType);
```

Comment:
I guess for completeness wouldn't we want to also log magCourtOutcome?

Reply:
added to log